dockerfiles/redist.txt

```diff
@@ -0,0 +1,97 @@
+This text file is the file (e.g., redist.txt, etc) specified in the accompanying
```

Comment:
remove this file

Reply:
@HectorSVC can you please review/approve this PR?

Reply:
removing